docs/DesignDocs/SGX_QuoteEx_Integration.md

```diff
@@ -0,0 +1,121 @@
+# Attestation: OE SDK Integration with Intel® SGX SDK quote-ex Library for Evidence Generation in New Formats
+
+This design document proposes an extension of the OE SDK implementation for integration with Intel® SGX SDK quote-ex library, for support of generation of evidence in new SGX formats such as Enhanced Privacy ID (EPID).
```

Comment:
grammar nit: insert "the" before "Intel"

Reply:
@dthaler thank you very much for for your proofreading.